### PR TITLE
Fix CI unit tests

### PR DESCRIPTION
**Pull request by machacekondra**
.. and remove duplicated workflow to run unit tests and integration tests

### CODE REVIEW

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -110,148 +110,13 @@ jobs:
           # https://github.com/ansible-community/ansible-test-gh-action#pull-request-change-detection
           pull-request-change-detection: false
 
-###
-# Unit tests (OPTIONAL)
```
Comment (bardielle) [inline .github/workflows/ansible-test.yml:114]: Does it mean that you removed the UT? there is not UT running at all after that PR?

Comment (machacekondra) [reply .github/workflows/ansible-test.yml:114]: Well, I removed duplicated UT, there are already UT running in `github/workflows/test.yml`

Comment (bardielle) [reply .github/workflows/ansible-test.yml:114]: Maybe we can change the file names? 
ansible-test and test are confusing names... maybe ansible-sanity.yml?

Comment (machacekondra) [reply .github/workflows/ansible-test.yml:114]: Yeah, that would be better.

**Pull request merged**